worker/src/handlers/post.ts

```diff
@@ -73,7 +92,22 @@ const sanitizePayload = (
     throw new Error("Missing required keys in the payload");
   }
 
-  if (!InstallationTypes.includes(payload.installation_type)) {
+  if (String(payload.uuid).length !== 32) {
+    throw new Error("Wrong UUID format");
+  }
+
+  const vs = payload.version.split(".");
```

Comment:
Should you validate that this is a string.

Reply:
Sure